pl_bolts/datamodules/imagenet_datamodule.py

```diff
@@ -7,23 +8,13 @@
 
 from pl_bolts.transforms.dataset_normalizations import imagenet_normalization
 
-try:
+_TORCHVISION_AVAILABLE = importlib.util.find_spec("torchvision") is not None
```

Comment:
lets move this to Bolts init so it is tested only once

Reply:
Sure. Does it sound reasonable to put these definitions in `pl_bolts/__init__.py`? Should I create a file like `pl_bolts/_imports.py` to define these variables?

Reply:
@Borda Would you mind checking #346?

Reply:
let's continue the talk there :]

Reply:
`pl_bolts/__init__.py` sounds as good place as it is used in any call to Bolts...